### DISCUSSION TITLE
Script license

### DISCUSSION THREAD

**Discussion by Amiralgaby** [thiggy01/ubuntu-20.04-change-gdm-background#21] [Q&A]
Hello,
I have seen your script and I have translated it into French in order to offer it to French users.
I have not seen a file explaining the license of the 2 scripts that allow the change of gdm background, I would like to translate them and maybe modify them to take into account zenity. I would not put any license too because I do not see the use of it, unless of course I ask you for one.

**Comment by thiggy01** [comment]
Since this is a simple script, I didn't think about any license.
The Idea is to help as many people as possible. So, you may translate and change it if you will.
